packages/flame_rive/pubspec.yaml

```diff
@@ -18,4 +17,5 @@ dev_dependencies:
   dartdoc: ^4.1.0
   flutter_test:
     sdk: flutter
+  dart_code_metrics: ^4.10.0
```

Comment:
This should not be re-added

Reply:
yeah, looks like it got left accidentally after a merge